src/pages/PlayPage.js

```diff
@@ -11,23 +12,39 @@ export default function PlayPage({
   playing,
   duration,
 }) {
+  useShortcut(['a'], onAnswer)
+  useShortcut(['1'], onAnswer)
+  useShortcut(['b'], onAnswer)
+  useShortcut(['2'], onAnswer)
+  useShortcut(['c'], onAnswer)
+  useShortcut(['3'], onAnswer)
```

Comment:
wdyt about `useShortcut(['a', '1', 'b', '2', 'c', '3'], onAnswer)`?

Reply:
thank you for the review, if i refactor it like this, you have to press all those keys together. 
In the next userstory i have to refactor this anyway. At the moment the shortcuts don't know witch button they fire.